### DISCUSSION TITLE
Could this migration guide page use an update?

### DISCUSSION THREAD

**Discussion by Josh68** [CMSgov/design-system#3049] [General]
https://github.com/CMSgov/design-system/edit/main/packages/docs/content/migration-guides/v7-spacing-updates.mdx

The `marginsBefore` image references "Padding," but I believe the example is supposed to be documenting only margin collapse and stacking behavior. Should the word "Padding" be replaced with "Spacing" in this image? (Not really the point, but also, should these examples even be images?)

**Comment by pwolfert** [comment]
Oh, you're right, I never noticed that. Darn it!

**Reply by Josh68** [reply]
No worries. Not a major thing, but since I noticed it, I thought I'd call it out.
